user-service/src/main/java/hcmus/alumni/userservice/controller/UserServiceController.java

```diff
@@ -353,4 +357,43 @@ public ResponseEntity<String> updateAlumniVerificationStatus(@PathVariable Strin
 		
 		return ResponseEntity.ok("Alumni verification approved successfully");
 	}
+    
+    @GetMapping("/count")
+	public ResponseEntity<Long> getSearchResultCount(@RequestParam(value = "status") String status) {
+		if (status.equals("")) {
+			ResponseEntity.status(HttpStatus.OK).body(0L);
+		}
+		return ResponseEntity.status(HttpStatus.OK).body(userRepository.getCountByStatus(status));
+	}
+
+	@GetMapping("")
+	public ResponseEntity<HashMap<String, Object>> getSearchResult(
+			@RequestParam(value = "page", required = false, defaultValue = "0") int page,
+			@RequestParam(value = "pageSize", required = false, defaultValue = "20") int pageSize,
+			@RequestParam(value = "fullName", required = false, defaultValue = "") String fullName,
+			@RequestParam(value = "status", required = false) String status,
```

Comment:
```suggestion
			@RequestParam(value = "status", required = false) Integer statusId,
```
lấy theo id của status lun